src/crimson/os/seastore/lba_manager/btree/btree_lba_manager.h

```diff
@@ -320,6 +325,7 @@ class BtreeLBAManager : public LBAManager {
     std::vector<alloc_mapping_info_t> alloc_infos;
     for (auto &extent : extents) {
       alloc_infos.emplace_back(alloc_mapping_info_t{
+	extent->get_laddr(),
```

Comment:
Sorry I'm not following here.

Looking at (should be) the only user: https://github.com/ceph/ceph/blob/adf4aca563c21106de1248a73fbef5a2bde5d7ab/src/crimson/os/seastore/transaction_manager.h#L359-L367

IIUC `extent->get_laddr()` here should always be NULL upon allocated by Cache, right?

So the new chance cannot be applied to any scenario?

Reply:
> So the new chance cannot be applied to any scenario?

Please see https://github.com/ceph/ceph/pull/57818#discussion_r1623926162